tests/apple-pay/ApplePayTestMockResponse.ts

```diff
@@ -0,0 +1,342 @@
+export const payments = `[{
+    "id": "applepay",
+    "gateway": null,
+    "logoUrl": "https:\/\/shariceapplauded-cloud-dev-vm.store.bcdev\/rHEAD\/modules\/checkout\/applepay\/images\/applepay-header@2x.png",
```

Comment:
Could we substitute data that's specific to your dev env with dummy data (e.g.: logoUrl, merchantId, sentry etc...)?